recipes/clingcon/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "clingcon" %}
+{% set version = "5.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/potassco/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: c6bd979b94eebc531a191d957feb53e2e4b37858c71f3f5e04d73ab50db96f43
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - make  # [unix]
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - clingo
+  run:
+    - clingo >=5.5
+    - python
+    - cffi  # [python_impl == 'cpython']
+
+test:
+  commands:
+    - clingcon --version
+    - python -m clingcon --version
```

Comment:
```suggestion
    - python -m clingcon --version
    - pip check
  requires:
    - pip
```

Reply:
I can add it but it wouldn't really do anything. When installing with cmake, only a minimal distinfo is put in place (assuming that conda is handling the dependencies correctly). Atm. setup.py is only used for deploying a pip package to `pypi.org` that is build very differently to avoid having to bundle shared libraries.

Reply:
I'll guess I add it because then at least the minimal metadata is checked.

Reply:
Yeah it’s also useful to avoid the chance of downstream packages failing for something which is a problem with the upstream metadata